Runtime/BacktraceClient.cs

```diff
@@ -170,6 +166,27 @@ public IEnumerable<string> GetAttachments()
             return _clientReportAttachments;
         }
 
+        /// <summary>
+        /// Set a client attribute that will be included in every report
+        /// </summary>
+        /// <param name="key">Attribute name</param>
+        /// <param name="value">Attribute value</param>
+        /// <returns>True, if the value was added. Otherwise false.</returns>
+        public bool SetAttribute(string key, string value) 
+        {
+            if (string.IsNullOrEmpty(key)) 
+            {
+                return false;
+            }
```

Comment:
Why not throw an exception here?

Reply:
I prefer to stay with the solution where we won't throw an error in case of a user mistake.